omero/developers/Python.rst

```diff
@@ -6,24 +6,57 @@ libraries.
 
 From OMERO 5.6.0 release, the client library ``omero-py`` supports Python 3 and
 is now available on PyPI_ and Conda_. The ``omero-py`` API documentation is available at https://omero-py.readthedocs.io/.
-We recommend you use a Python virtual environment to install the client library. You can create one using either ``venv`` or ``conda`` (preferred).
-If you opt for Conda_, you will need
-to install it first, see `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ for more details.
-
-To install ``omero-py`` using venv:
+We recommend you use a Python virtual environment to install the client library. You can create one using either ``venv``, ``conda`` or ``mamba`` .
```

Comment:
Space after mamba. We don't specify a preferred tool now? (Which is fine, just checking that's intentional)?

Reply:
that's intentional